source/configuration/read.txt

```diff
@@ -133,6 +133,10 @@ You can configure the following properties to read from MongoDB:
 Partitioner Configurations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Partitioners change the read behavior for batch reads with the {+driver-short+}.
+They have no effect on structured streams because structured streams only
```

Comment:
Issue:

From reading the [Spark Structured Streaming Overview,](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html) it seems like that's a feature name and should be capitalized (similar to how it is on the https://www.mongodb.com/docs/spark-connector/current/structured-streaming/) page.

Suggestion:

Capitalize the product name and describe which part of it produces a single stream (perhaps the "[processing engine]( processing engine)"?).

Similar feedback applies to the mention in the other changed file.

Suggestion 2:

I think to make this more compact, "no effect on" could be replaced with "do not affect". Could probably omit "only" as well since that's implied by the adjective "single".



```suggestion
They do not affect Structured Streams because the Structured Streams processing engine 
```